File: Python/PRAK601-2210817220009-Azka Amalia.py
baris, kolom = map(int,input().split())
angka = [[0 for row in range(0,kolom)]for col in range(0,baris)]
for i in range(0,baris):
    for j in range(0,kolom):
        num=int(input())
        angka[i][j]=num
for i in range(0,baris):
    for j in range(0,kolom):
        print(angka[i][j],end=' ')
    print()      

File: C/PRAK601-2210817220009-Azka Amalia.c
#include<stdio.h>
int main()
{
    int angka[20][20];
    int baris, kolom, i, j;
    scanf("%d", &baris);scanf("%d", &kolom);
    for(i=0;i<baris;i++){
        for(j=0;j<kolom;j++){
            scanf("%d", &angka[i][j]);}}
    printf("\n");
    for(i=0;i<baris;i++){
        for(j=0;j<kolom;j++){
            printf("%d ", angka[i][j]);}
        printf("\n");}
}
